tests/commands/rhsm.py

```diff
@@ -0,0 +1,77 @@
+#
+# Martin Kolman <mkolman@redhat.com>
```

Comment:
No need for names anymore, that's what git blame is for :)

Reply:
OK, removed. :)